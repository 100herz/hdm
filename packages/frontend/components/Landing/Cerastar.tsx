import styled from '@emotion/styled'
import dynamic from 'next/dynamic'

export default function Cerastar({ className = '' }: React.HTMLAttributes<HTMLElement>) {
  const Pdf = dynamic(() => import('@/components/Utils/Pdf'), { ssr: false })

  return (
    <section
      id="cerastar"
      className={`container relative lg:grid lg:grid-cols-2 lg:gap-32 mt-32 pt-20 pb-24 lg:mt-24 ${className}`}
    >
      <BackgroundImage />

      <header>
        <div className="lg:ml-32">
          <h2 className="text-4xl font-bold lg:text-5xl">Cerastar</h2>
          <p className="mt-4 text-xl leading-snug lg:text-2xl">
            Der neue Star unter <br />
            den Bodenbelägen
          </p>
        </div>

        <picture className="block mt-16 -mx-6 lg:-ml-20 lg:-mr-8">
          <source srcSet="/images/hdm-cerastar.webp" type="image/webp" />
          <source srcSet="/images/hdm-cerastar.png" type="image/png" />
          <img src="/images/hdm-cerastar.png" alt="Cerastar - der neue Star unten den Bodenbelägen" />
        </picture>
      </header>

      <div className="lg:mt-32 lg:pr-16">
        <h3 className="mt-12 font-bold lg:text-lg">
          Cerastar heißt der neue Designbodenbelag mit keramischer Nanocoat Oberflächenversiegelung.
        </h3>
        <p className="mt-8 lg:text-lg">
          1,2 Mio. feinste Keramikpartikel pro Quadratzentimeter machen die Oberfläche von Cerastar – gegenüber
          Oberflächen vergleichbarer Bodenbeläge – bis zu dreimal kratzfester und strapazierfähiger.
        </p>
        <p className="mt-8 lg:text-lg">
          So entsteht eine äußerst abriebfeste Schutzschicht, die deutlich resistenter gegen Mikrokratzer ist. Zugleich
          verleiht die hochtransparente Versiegelung den Dekoren eine einzigartige brillante Farbtiefe und eleganten
          Glanz.
        </p>
        <p className="mt-8 lg:text-lg">
          Weitere Informationen finden Sie unter{' '}
          <a className="underline text-hdm hover:text-hdm-dark" href="https://cerastar.eu">
            cerastar.eu
          </a>
          .
        </p>

        <div className="mt-16 text-center lg:text-left lg:mt-24">
          <Pdf title="Cerastar Katalog" filename="/documents/hdm-cerastar.pdf" />
        </div>
      </div>
    </section>
  )
}

const BackgroundImage = styled.div`
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: -2;
  overflow-x: hidden;
  background-color: #f8efef;

  &::before {
    position: absolute;
    top: 3rem;
    left: 45%;
    z-index: -1;
    width: 581px;
    height: 409px;
    background: url(/images/hdm-cerastar-muster.png) no-repeat;
    opacity: 0.7;
    content: '';
  }

  @media (min-width: 1024px) {
    &::before {
      top: 5rem;
    }
  }
`
