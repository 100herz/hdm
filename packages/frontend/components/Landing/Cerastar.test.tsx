import { render } from '@/test/test-utils'

import Cerastar from './Cerastar'

describe('Cerastar component', () => {
  it('matches snapshot', () => {
    const { container, getByText } = render(<Cerastar />)
    expect(getByText('Cerastar')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
