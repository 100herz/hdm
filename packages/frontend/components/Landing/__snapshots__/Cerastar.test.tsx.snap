// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cerastar component matches snapshot 1`] = `
.emotion-0 {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: -2;
  overflow-x: hidden;
  background-color: #f8efef;
}

.emotion-0::before {
  position: absolute;
  top: 3rem;
  left: 45%;
  z-index: -1;
  width: 581px;
  height: 409px;
  background: url(/images/hdm-cerastar-muster.png) no-repeat;
  opacity: 0.7;
  content: '';
}

@media (min-width: 1024px) {
  .emotion-0::before {
    top: 5rem;
  }
}

<section
  class="container relative lg:grid lg:grid-cols-2 lg:gap-32 mt-32 pt-20 pb-24 lg:mt-24 "
  id="cerastar"
>
  <div
    class="emotion-0 emotion-1"
  />
  <header>
    <div
      class="lg:ml-32"
    >
      <h2
        class="text-4xl font-bold lg:text-5xl"
      >
        Cerastar
      </h2>
      <p
        class="mt-4 text-xl leading-snug lg:text-2xl"
      >
        Der neue Star unter 
        <br />
        den Bodenbelägen
      </p>
    </div>
    <div
      class="mt-16 -mx-6 lg:-ml-20 lg:-mr-8"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            role="presentation"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzY4IiBoZWlnaHQ9IjcxNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="Cerastar - der neue Star unten den Bodenbelägen"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </div>
    </div>
  </header>
  <div
    class="lg:mt-32 lg:pr-16"
  >
    <h3
      class="mt-12 font-bold lg:text-lg"
    >
      Cerastar heißt der neue Designbodenbelag mit keramischer Nanocoat Oberflächenversiegelung.
    </h3>
    <p
      class="mt-8 lg:text-lg"
    >
      1,2 Mio. feinste Keramikpartikel pro Quadratzentimeter machen die Oberfläche von Cerastar – gegenüber Oberflächen vergleichbarer Bodenbeläge – bis zu dreimal kratzfester und strapazierfähiger.
    </p>
    <p
      class="mt-8 lg:text-lg"
    >
      So entsteht eine äußerst abriebfeste Schutzschicht, die deutlich resistenter gegen Mikrokratzer ist. Zugleich verleiht die hochtransparente Versiegelung den Dekoren eine einzigartige brillante Farbtiefe und eleganten Glanz.
    </p>
    <p
      class="mt-8 lg:text-lg"
    >
      Weitere Informationen finden Sie unter
       
      <a
        class="underline text-hdm hover:text-hdm-dark"
        href="https://cerastar.eu"
      >
        cerastar.eu
      </a>
      .
    </p>
    <div
      class="mt-16 text-center lg:text-left lg:mt-24"
    />
  </div>
</section>
`;
